packages/block-editor/src/components/link-control/style.scss

```diff
@@ -171,6 +181,34 @@
 	}
 }
 
+.block-editor-link-control__loading {
+	margin: $grid-size-large; // when only loading control is shown it requires it's own spacing.
+	display: flex;
+	align-items: center;
+
+	.components-spinner {
+		margin-top: 0;
+	}
+}
+
+// Separate Create button when following other suggestions.
+.components-button + .block-editor-link-control__search-create {
+	margin-top: 20px;
+	overflow: visible;
+
+	// Create fake border. We cannot use border because the button has a border
+	// radius applied to it
+	&::before {
+		content: "";
+		position: absolute;
+		top: -10px;
+		left: 0;
+		display: block;
+		width: 100%;
+		border-top: 1px solid #ccc;
```

Comment:
Should we be using one of the standard colors here, rather than `#ccc` ?

https://github.com/WordPress/gutenberg/blob/056d7c108ac968363b7a747660ea50b13069f87a/packages/base-styles/_colors.scss#L7-L25

Reply:
I'm sorry. This was very lax of me. I have amended.

@shaunandrews I wonder whether we need to correct all the gray references in this style sheet to be one of the normalise variables as shown above?

Reply:
Definitely! I can update the #ccc to the closest variable, and can normalize the rest if @shaunandrews thinks it'll be best. Sounds reasonable to me.

Reply:
I updated all the colors throughout this stylesheet to refer to the closest `base-styles/_colors.scss` reference. In one case, I did use a darker color (`$dark-gray-300`) for a text item that had a lighter color to make it meet AA standards.

Reply:
Thank you for your help here @jeryj. Very much appreciated.